packages/sdk/src/machines/utils.ts

```diff
@@ -20,6 +20,13 @@ export const SUPPORTED_MINTER_TYPES = [
   Minter_Type_Names_Enum.MinterDaExpSettlementV3,
 ];
 
+export const SUPPORTED_SETTLEMENT_CLAIM_MINTER_TYPES = [
+  Minter_Type_Names_Enum.MinterDaExpSettlementV0,
```

Comment:
I'm pretty sure we never deployed a V0 minter, but fine to leave here if you want

Reply:
Yeah, I just figured I'd include all of them because they have the same ABI for the relevant functions